docs/ucx/docs/process/index.mdx

```diff
@@ -8,7 +8,8 @@ On a high level, the steps in migration process are:
 1. [assessment](/docs/reference/workflows#assessment-workflow)
 2. [group migration](/docs/reference/workflows#group-migration-workflow)
 3. [table migration](/docs/process/#table-migration-process)
-4. [data reconciliation](/docs/reference/workflows#post-migration-data-reconciliation-workflow)
+4. [HMS Federation](/docs/process/#hms-federation)
```

Comment:
@FastLee : This is not a next step after table migration it is an alternative to table migration, could you make that clear?